CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ the code was deployed.
 - "doorMissedFrequently" field to JSON data submitted by brave sensors that is sent to /api/heartbeat (CU-860rk8v2a)
 - Sentry log in the case that doorMissedFrequently is true in posted data from brave sensor (CU-860rk8v2a)
 - GitHub Actions to deploy to AWS infrastructure on Dev and Staging (CU-860ra8f7q).
+- Unit tests for API authorize function (CU-8678uuvjm)
```

Comment:
So now that we've done a deployment and created a new section for 9.7.0, there will be a conflict in this file when we try to merge with `main` and we'll want to make sure that it is resolved correctly. 

What I normally do is I will rebase off the new `main` branch. Which means, on my local, I would:
1. Checkout main `git checkout main`
2. Pull the latest `git pull`
3. Checkout my branch again `git checkout mdb-investigate-api-authorize`
4. Rebase off main `git rebase main`

This will print out a bunch of words that basically say that you have a conflict and it will let you know that `CHANGELOG.md` is one of the files that has a conflict (hopefully the only file, in this case). Then you:

5. Resolve the conflict in `CHANGELOG.md`. In this case, you'll probably need to move your description of what changed in this branch from the 9.7.0 section into the Unreleased section
6. Continue your rebase `git rebase --continue`
7. You may need to repeat steps 5 and 6 a couple of times

Now when you run `git status` you'll see that it thinks that you are X commits ahead and Y commits behind origin. That's expected because by using `git rebase`, you've gone back and changed history. We like this new world that we're in better than the old one, so we want to push it. But GitHub has protections so that you don't change history by accident. So you'll need to:

7: Force push `git push -f`

Reply:
Gotcha!

Reply:
Looks like this will need to be done again because Tara merged her change which added something to the `CHANGELOG.md`. This is going to be a frequent occurrence with multiple people working on the same code base.